inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -38,8 +38,8 @@ interface ICanvasAxis {
 }
 
 export interface IActionableKeyFrame {
-  frame: number;
-  value: any;
+  frame?: number | string;

```

Comment:
Why would a frame be a string here ?

Reply:
These properties are the ones connected to the input box. So this is for the case of an empty/disabled input box, so we can have an empty value, if not we force the input to always have a 0.

Reply:
this is already number | undefined with the question mark. So I think you might not need the string if undefined is used in case of empty or invalid string ?

Reply:
There is an update I'll do to this after a review today.